e2etest/bgp_test.go

```diff
@@ -93,32 +93,42 @@ var _ = ginkgo.Describe("BGP", func() {
 			},
 		},
 	}
-	ginkgo.BeforeEach(func() {
-		var err error
-		if containersNetwork == "host" {
-			if net.ParseIP(hostIPv4) == nil {
-				framework.Fail("Invalid hostIPv4")
-			}
-			if net.ParseIP(hostIPv6) == nil {
-				framework.Fail("Invalid hostIPv6")
+
+	ginkgo.AfterEach(func() {
```

Comment:
Is there a guarantee that the first AfterEach is always called before the second one?

Reply:
Yes. Ginkgo parses them in order.
That's also the reason why I moved the framework initialization, otherwise it would have cleared the namespace before we do the dump.